### DISCUSSION TITLE
How to fix 404 error?

### DISCUSSION THREAD

**Discussion by Tupu4545** [FlareSolverr/FlareSolverr#1057] [Q&A]
I have setup everything correctly however I still get this error:

Unable to connect to indexer, please check your DNS settings and ensure IPv6 is working or disabled. The proxy tunnel request to proxy 'http://60.0.0.100:8191/' failed with status code '404'."

**Comment by ilike2burnthing** [comment]
Closing as Prowlarr issue. I suggesting googling the error, and if you can't find a solution from there, reach out to their Discord server for support.
